client-app-example/public/client-app-1/index.html

```diff
@@ -8,21 +8,37 @@ <h1>Application #1</h1>
     <h2>Routing</h2>
     <h3>Current Local Route:</h3>
     <div id="path"></div>
-    <h3>Current Host Route:</h3>
-    <div id="hostPath"></div>
+    <h3>Current Host Route Calculated From Client:</h3>
```

Comment:
I will go back and clean up some of these client app examples so it better showcases how to generate links, especially using the new custom elements `ifc-client-link` and `ifc-host-link`